lua/minigames/engine/sh_convars.lua

```diff
@@ -143,9 +150,62 @@ else
 			local mg = mgs[i]
 			local cvd = mg.conVarData
 
-			if not istable(cvd) or table.Count(cvd) < 1 then continue end
+			local size = 20
+
+			if not istable(cvd) or table.Count(cvd) < 1 then
```

Comment:
That`s copy & paste from here and looks really inconsistent: https://github.com/TTT-2/ttt2mg/pull/4/files#diff-fd97296f5e50bccad863dde7d14e8c84R210